src/components/FloatingBox/BookingButton.tsx

```
@@ -4,19 +4,33 @@ interface BookingButtonProps {
   return (
     <button
       onClick={onClick}
-      disabled={disabled}
-      className={`mt-4 mb-6 w-full max-w-sm rounded-lg py-20 font-medium ${disabled ? 'cursor-not-allowed bg-gray-300 text-gray-500' : 'bg-green-800 text-white hover:bg-green-900'}`}
+      disabled={disabled || onBooking}
+      className={`relative mt-4 mb-6 w-full max-w-sm rounded-lg py-10 font-medium transition-colors ${
+        disabled || onBooking
+          ? 'cursor-not-allowed bg-gray-300 text-gray-500'
+          : 'bg-green-800 text-white hover:bg-green-900'
+      }`}
     >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)